TheFourFifths/consus
# Fix continuous deployment

### Summary

I guess I fixed continuous deployment. All I was set my NPM API key & email.
I also upgraded some dependencies.

### Checklist


### How to Review

1. Ensure there's a git tag `v0.5.1`
1. Ensure the v0.5.1 is released at https://www.npmjs.com/package/consus

### Links

_None_
